.github/workflows/push.yml

```diff
@@ -13,37 +13,25 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: ./.github/actions/stripe_setup
-      - name: Generate Android release bundle for Identity example app
-        run: ./gradlew :identity-example:bundleRelease
-      - name: Upload artifact to Emerge
-        uses: EmergeTools/emerge-upload-action@3477b597fc62054136eb6f499e0ba78144f8a999
-        with:
-          artifact_path: identity-example/build/outputs/bundle/theme1Release/identity-example-theme1-release.aab
-          emerge_api_key: ${{ secrets.EMERGE_API_KEY }}
-          build_type: push
+      - name: Upload Identity example release bundle to Emerge
+        run: ./gradlew :identity-example:emergeUploadReleaseAab
+        env:
+          EMERGE_API_KEY: ${{ secrets.EMERGE_API_KEY }}
   upload-financial-connections:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
       - uses: ./.github/actions/stripe_setup
-      - name: Generate Android release bundle for Financial Connections example app
-        run: ./gradlew :financial-connections-example:bundleRelease
-      - name: Upload artifact to Emerge
-        uses: EmergeTools/emerge-upload-action@3477b597fc62054136eb6f499e0ba78144f8a999
-        with:
-          artifact_path: financial-connections-example/build/outputs/bundle/release/financial-connections-example-release.aab
-          emerge_api_key: ${{ secrets.EMERGE_API_KEY }}
-          build_type: push
+      - name: Upload Financial Connections example release bundle to Emerge
+        run: ./gradlew :financial-connections-example:emergeUploadReleaseAab
+        env:
+          EMERGE_API_KEY: ${{ secrets.EMERGE_API_KEY }}
   upload-payment-sheet:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
       - uses: ./.github/actions/stripe_setup
-      - name: Generate Android release bundle for Payment Sheet example app
-        run: ./gradlew :paymentsheet-example:bundleRelease
-      - name: Upload artifact to Emerge
-        uses: EmergeTools/emerge-upload-action@3477b597fc62054136eb6f499e0ba78144f8a999
-        with:
-          artifact_path: paymentsheet-example/build/outputs/bundle/release/paymentsheet-example-release.aab
-          emerge_api_key: ${{ secrets.EMERGE_API_KEY }}
-          build_type: push
+      - name: Upload Financial Connections example release bundle to Emerge
```

Comment:
I think the name on this one might be incorrect -- it says "Financial connections" but the name of this step is for payment sheet

Reply:
Updated!